### PR TITLE
update autocomplete to use facetSpell service

### PR DESCRIPTION
**Pull request by JrgenGNielsen**

### CODE REVIEW

File: includes/ting_search_autocomplete.admin.inc [hunk 1]
```diff
@@ -53,6 +53,7 @@ function ting_search_autocomplete_admin() {
     '#title' => t('Facet Index', array(), array('context' => 'ting_search_autocomplete')),
     '#size' => 25,
     '#default_value' => $default_settings['facetIndex'],
+    '#required' => TRUE,
```
Comment (kasperg) [inline includes/ting_search_autocomplete.admin.inc]: Is this related to the change at hand?

Comment (JrgenGNielsen) [inline includes/ting_search_autocomplete.admin.inc]: just so - the facetspell method has index and facetIndex as required parameters. See also: [http://testbroend.dbc.dk:7080/suggest/#facetspell]

**Comment by kasperg** [comment]
Reviewed and approved. I have a question but assume this is OK.

File: includes/ting_search_autocomplete.admin.inc (hunk 1 above)
Comment (kasperg) [inline includes/ting_search_autocomplete.admin.inc]: I can access the service :\.

Comment (JrgenGNielsen) [inline includes/ting_search_autocomplete.admin.inc]: ah, *duh* - I'll go bother the backend team for an external accessible service
